README.adoc

```diff
@@ -120,7 +121,7 @@ The Java EE Security specification also defines a common SecurityContext API tha
 
 The `doGet()` method uses the `securityContext.isCallerInRole()` API to determine the current user's role  and forward the response to a proper page.
 
-Open the `src/main/webapp/WEB-INF/web.xml` file:
+Open the `src/main/webapp/WEB-INF/web.xml` file to view how the JSF pages are secured:
```

Comment:
Probably not "JSF pages" but "web pages" to be more generic?

Reply:
Okay! Noted.